content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -1,7 +1,110 @@
+and actively manages every object's actual state to match the desired state you
+supplied.
+
+For example: in Kubernetes, a Deployment is an object that can represent an
+application running on your cluster. When you create the Deployment, you
+might set the Deployment `spec` to specify that you want three replicas of
+the application to be running. The Kubernetes system reads the Deployment
+spec and starts three instances of your desired application—updating
+the status to match your spec.If any of those instances should fail
+(a status change), the Kubernetes system responds to the difference
+between spec and status by making a correction (in this case, starting
+a replacement instance).
+
+## Describing a Kubernetes object
+
+When you create an object in Kubernetes, you must provide the object spec that describes its desired state, as well as some basic information about the object (such as a name). When you use the Kubernetes API to create the object (either directly or via `kubectl`), that API request must include that information as JSON in the request body. **Most often, you provide the information to `kubectl` in a .yaml file.** `kubectl` converts the information to JSON when making the API request.
+
+Here's an example `.yaml` file that shows the required fields and object spec for a Kubernetes Deployment:
+
+{{< codenew file="application/deployment.yaml" >}}
+
+One way to create a Deployment using a `.yaml` file like the one above is to use the
+[`kubectl apply`](/docs/reference/generated/kubectl/kubectl-commands#apply) command
+in the `kubectl` command-line interface, passing the path to the `.yaml` manifest as
+an argument. Here's an example:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/deployment.yaml
+```
+
+The output is similar to this:
+
+```
+deployment.apps/nginx-deployment created
+```
+
+### Required fields
+
+In the manifest for the Kubernetes object you want to create, you'll need to set values for the following fields:
+
+* `apiVersion` - Which version of the Kubernetes API you're using to create this object
+* `kind` - What kind of object you want to create
+* `metadata` - Data that helps uniquely identify the object, including a `name` string, `UID`, and optional `namespace`
+* `spec` - What state you desire for the object
+
+The precise format of the object `spec` is different for every Kubernetes object, and contains nested fields specific to that object. The [Kubernetes API Reference](https://kubernetes.io/docs/reference/kubernetes-api/) can help you find the spec format for all of the objects you can create using Kubernetes.
+
+For example, the reference for Pod details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
+for a Pod in the API, and the reference for Deployment details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec) for Deployments.
```

Comment:
Lines 88-90, do these sentences add significant value?

Reply:
The rebase should update these.